overview/glossary.md

```diff
@@ -0,0 +1,42 @@
+| [muPort-DID-Resolver](https://github.com/uport-project/muport-did-resolver)    | A DID resolver for muPort/SimpleID resolution with support for creating identities.                                                                                              |
+| PKI                      | Public Key Infrastructure.  uPort implements a simple PKI to verify off-chain messages                                                                                                                                               |
+| Public Key               |  Used to verify privately signed data                                                                                                                                                                                                |
+| Private Key              |  Used to sign data and transactions                                                                                                                                                                                                  |
+| Recovery Seed            |  A 12-word pass phrase used to retrieve the private keys of an Identity                                                                                                                                                              |
+| [Secp256k1-DID-Resolver](https://github.com/uport-project/secp256k1-did-resolver) | A DID resolver that uses secp256k1 public keys as decentralized identifiers, as oppose to an ethereum address.                                                                                     |
+| Selective Disclosure     | A request and response for specific data about claims (attestations) related to an decentralized identity.                                                                                                                            |
+| Segregated Account       | A key derived from the seed of a root identity to be used instead of the root key to avoid correlating blockchain activity with the root key.                                                                                        |
+| Signing                  | The act of using a private key to sign data and transactions.                                                                                                                                                                        |
+| Signing Key              | Also referred to as a Private key                                                                                                                                                                                                    |
+| Testnet                  | For uPort's purposes this is Rinkeby.                                                                                                                                                                                                |
```

Comment:
Ethereum test networks used by developers for testing out their applications. uPort supports both the Rinkeby and Kovan test networks (include links to them)